tests/db/test_bigchain_api.py

```diff
@@ -62,7 +41,7 @@ def test_get_spent_with_double_spend_detected(self, b, alice):
             b.get_spent(tx.id, 0)
 
     @pytest.mark.tendermint
-    def test_get_block_status_for_tx_with_double_inclusion(self, b, alice):
+    def test_double_inclusion(self, b, alice):
```

Comment:
Could you, please, also update the test below (`test_text_search`)?

We need to replace

```
# this query only works with MongoDB
try:
    assets = list(b.text_search('bigchaindb'))
except OperationError as exc:
    assert not isinstance(b.connection, LocalMongoDBConnection)
else:
    assert len(assets) == 3
```
with
```
assets = list(b.text_search('bigchaindb'))
assert len(assets) == 3
```
Using `LocalMongoDBConnection` is part of the test setup, we should not assert for that.